simulation/simulation_wrappers/eor_simulation_suite_enterprise.pro

```diff
@@ -9,18 +9,23 @@ pro eor_simulation_suite_enterprise, uvf_input = uvf_input, $
   if n_elements(flat_sigma) eq 0 then flat_sigma = 1
   if keyword_set(flat_sigma) then sim_in = 'flat' else sim_in = 'eor'
 
+  if keyword_set(large_healpix) then begin
+    restrict_hpx_inds='EoR0_high_healpix_inds_3x.idlsave'
+  endif
+
   if keyword_set(file_uvw) then begin
     if n_elements(run_name) eq 0 then run_name = 'mwa_zenith'
 
     version = 'bjh_sim_' + sim_in + '_' + run_name
 
     n_samples = 1
     no_suite_plots = 1
+    use_weight_cutoff_sim = 1
   endif else begin
     ;sample_factors = [.0001,.0002,.0005,.001,.005,.01,.05,.1,.5, 1]
-    sample_factors = [.0001,.0005,.001,.005,.01,.05,.1,.5, 1, 5]
+    sample_factors = [.0001,.0005,.001,.005,.01,.05,.1,.5, 1];, 5]
```

Comment:
Why not delete the commented-out code?

Reply:
yep, I can. The reason for the commenting is because higher values would be nice to run but I keep running out of memory. I'll clean it up and add a comment.